Sources/KituraNet/ClientRequest.swift

```diff
@@ -380,6 +380,10 @@ public class ClientRequest {
             self.port = isHTTPS ? 443 : 80
         }
 
+        defer {
+            try! group.syncShutdownGracefully()
```

Comment:
Are we sure we want to crash the app if the threads don't shut down (for whatever reason)? Is there an argument for carrying on?

```swift
do {
    try group.syncShutdownGracefully()
} catch {
    // meh
}

Reply:
I think we should just log a diagnostic message and carry on here. We don't want to crash.